assets/bingo/lang/de_de.json

```diff
@@ -1,17 +1,286 @@
+// cls; pushd C:\github\bingo\assets\bingo\lang; $en = (gc .\en_us.json | ConvertFrom-Json -AsHashtable); $de = (gc .\de_de.json | ConvertFrom-Json -AsHashtable); popd;  $missing = @( $en.Keys | ? {-not $de.Keys.Contains($_) } ); $missing.Count; $missing | sort
 {
+	"bingo.game.go_to_lobby": "%s um zur Lobby zurückzukehren.",
+	"bingo.game.go_to_lobby.hint": "Hinweis: Du kannst jederzeit mit dem Befehl %s zur Lobby zurückkehren",
+	"bingo.game.go_to_lobby.click": "Hier klicken",
+
+	"bingo.lobby.machinery.sign.line1": "Maschinenraum",
+	"bingo.lobby.machinery.sign.line2": "Zugang nur für",
+	"bingo.lobby.machinery.sign.line3": "autorisiertes ",
+	"bingo.lobby.machinery.sign.line4": "Personal",
+
+	"bingo.lobby.credits.neun_einser.sign.line2": "",
+	"bingo.lobby.credits.neun_einser.sign.line3": "Alles von Minecraft",
+	"bingo.lobby.credits.neun_einser.sign.line4": "Bingo v5",
+
+	"bingo.lobby.credits.amber_wat.sign.line2": "",
+	"bingo.lobby.credits.amber_wat.sign.line3": "Negative Space",
+	"bingo.lobby.credits.amber_wat.sign.line4": "Font",
```

Comment:
I am not sure too. Is this meant as the name or the description? If it is the name we should not translate it. If it is the description we should use the translation.

Reply:
The name of the pack is NegativeSpaceFont, but that's also the description 🤔

Reply:
I think it would be better to treat it as description here -- If you click the sign you get a github link (and also I give attribution on the release page), so in terms of attribution this should be fine either way.

Reply:
While "Schriftart für negative Leerzeichen" is technichally correct I think "Schriftart für negativen Raum" captures the spirit of the font better. What translation do you prefer?

Reply:
Hmm, I dunno, it does talk about space as in the space character rather than the general term space (also if you'd explain on how the card display works, you'd say "it uses negative spaces" (also from the description on https://github.com/AmberWat/NegativeSpaceFont/releases “This resource pack for Minecraft Java Edition contains a font with a lot of negative and positive size spaces.”

A plural form of space does in this context only make sense if you talk about the character.

So the thing described here literally translated is a "Leerzeichen" and in my opinion, that is the term that makes more sense here and is more specific about what it actually is.